web_src/js/features/admin/common.js

```diff
@@ -204,22 +205,18 @@ export function initAdminCommon() {
           break;
       }
     });
-    $('#delete-selection').on('click', function (e) {
+    $('#delete-selection').on('click', async function (e) {
       e.preventDefault();
       const $this = $(this);
       $this.addClass('loading disabled');
-      const ids = [];
+      const data = new FormData();
       $checkboxes.each(function () {
         if ($(this).checkbox('is checked')) {
-          ids.push($(this).data('id'));
+          data.append('ids[]', $(this).data('id'));
```

Comment:
?
That looks really weird.
Are you sure that's the correct key?

Reply:
Yes, you can see it in your devtools

Reply:
My question was rather:
The backend accepts and wants this specific key?
It does look quite strange to me…

Reply:
It does
https://github.com/go-gitea/gitea/blob/f9207b09479df964872d68842469991042b5497f/routers/web/admin/notice.go#L50